net/zmap/Portfile

```diff
@@ -1,10 +1,10 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           cmake 1.0
+PortGroup           cmake 1.1
 PortGroup           github 1.0
 
-github.setup        zmap zmap 3.0.0-beta1 v
+github.setup        zmap zmap 3.0.0-RC2 v
 
 categories          net
 license             apache-2
```

Comment:
this should be capitalized

Reply:
@reneeotten To make sure I am understanding you correctly, do you mean that:

`license apache-2`

Should instead be:

`license Apache-2`

?